woocommerce/woocommerce-android
# Business logic to open order detail on order item click in 2 pane layout

Closes: #10588 


### Description

This PR introduces business logic to open the selected order in the detail pane

Note: This PR does not include highlighting the selected order in the order list screen

### Testing instructions

1. Run the app on physical tablet or in a tablet emulator
2. Click on the `Orders` tab
3. Select order items from the order listing screen on the left
4. Ensure the selected order's details are displayed on the detail pane on the right


### Images/gif

[pr_10592.webm](https://github.com/woocommerce/woocommerce-android/assets/1331230/5b5ed4fb-eab6-49fb-9e90-89628af3ddb1)
